test/utils/cyaHelpers/item/additionalInformation/AdditionalInformationAnswersHelperSpec.scala

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils.cyaHelpers.item.additionalInformation
+
+import base.SpecBase
+import controllers.item.additionalInformation.index.routes
+import generators.Generators
+import models.{Index, Mode}
+import org.scalacheck.Arbitrary.arbitrary
+import org.scalacheck.Gen
+import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
+import pages.item.AddAdditionalInformationYesNoPage
+import pages.item.additionalInformation.index.{AdditionalInformationPage, AdditionalInformationTypePage}
+import viewmodels.ListItem
+
+class AdditionalInformationAnswersHelperSpec extends SpecBase with ScalaCheckPropertyChecks with Generators {
+
+  "AdditionalInformationAnswersHelper" - {
+
+    "listItems" - {
+
+      "when empty user answers" - {
+        "must return empty list of list items" in {
+          forAll(arbitrary[Mode]) {
+            mode =>
+              val userAnswers = emptyUserAnswers
+
+              val helper = new AdditionalInformationAnswersHelper(userAnswers, mode, itemIndex)
+              helper.listItems mustBe Nil
+          }
+        }
+      }
+
+      "when user answers populated with complete additional information entries" in {
```

Comment:
Worth adding a test for an in progress one as well?

Reply:
I think we added one in for that on Friday when we were doing the progressive disclosure approach and I've most likely deleted it when I chopped the head off the commits to revert back to the original version! My bad